rotkehlchen/types.py

```diff
@@ -508,6 +508,13 @@ def to_range_prefix(self, range_type: Literal['txs', 'internaltxs', 'tokentxs'])
         """Provide the appropriate range prefix for the DB for this chain"""
         return f'{self.value}{range_type}'
 
+    def to_location(self) -> 'Location':
```

Comment:
as I mentioned earlier in the PR this change is not needed if you reorganize how the protocol balance class is initialized. Use chainID there.